minio/minio
# metrics: add replication metrics on proxied requests

## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description
dependency https://github.com/minio/madmin-go/pull/268

## Motivation and Context


## How to test this PR?
set up bucket/site replication and see prometheus metrics for `mc stat` or `mc head` on non-existent object/version - the metrics count should reflect proxied requests.

## Types of changes

## Checklist:
